ui/package.json

```diff
@@ -17,8 +17,10 @@
   },
   "scripts": {
     "start": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:dev",
-    "start:docker": "yarn generate && yarn build:vendor && yarn run start:dev",
+    "start:cloud": "yarn install && yarn generate && cross-env TS_NODE_PROJECT=\"webpack.tsconfig.json\" && yarn run build:vendor && yarn run start:cloud-dev",
     "start:dev": "webpack-dev-server --config ./webpack.dev.ts",
+    "start:cloud-dev": "CLOUD=true CLOUD_LOGOUT_URL=http://localhost:8080/api/v2/signout CLOUD_URL=http://localhost:9999 webpack-dev-server --config ./webpack.dev.ts",
```

Comment:
thoughts on `dev-cloud` vs `cloud-dev`?

I'm thinking

`start:dev`
`start:dev-cloud`
`start:dev-robot_dragon`

might be very logical and linear